polaris-react/src/components/Columns/Columns.tsx

```diff
@@ -6,22 +6,32 @@ import type {
 
 import {
   getResponsiveProps,
+  getResponsiveValue,
   sanitizeCustomProperties,
+  ResponsiveValue,
 } from '../../utilities/css';
 import type {ResponsiveProp} from '../../utilities/css';
 
 import styles from './Columns.scss';
 
-type Columns = {
-  [Breakpoint in BreakpointsAlias]?: number | string;
-};
+type ColumnWidths = 'oneThird' | 'oneHalf' | 'twoThirds';
+type ColumnTypes = number | string | ColumnWidths[];
```

Comment:
```suggestion
type ColumnType = number | string | ColumnWidths[];
```

Light suggestion to make this singular.